styles/dark.css

```
@@ -0,0 +1,857 @@
+          color: var(--color-text-trait);
+        }
+      }
+      section.major > header a,
+      section.major > .subsection > header a {
+        color: var(--text-light);
+      }
+
+      > .tab.active.character .subsection {
+        &.details .abcd .detail h3 {
+          color: var(--text-light);
+          background-color: var(--color-bd-text-input);
+        }
+
+        &.speeds .speed {
+          .label {
+            color: var(--text-light);
+          }
+
+          .value.rollable svg path {
+            fill: var(--text-light);
+          }
+
+          .value,
+          .value i {
+            color: var(--text-light);
+            fill: var(--text-light);
+          }
+
+          .value:hover,
+          .value i:hover {
+            filter: drop-shadow(0 0 1px var(--color-shadow-dark)) drop-shadow(0 0 2px var(--color-text-dark-4)) drop-shadow(0 0 3px var(--color-text-light-3));
+          }
+        }
+
+        &.attributes ul {
+          .label {
+            color: var(--text-light);
+          }
+
+          h3.modifier {
+            border-image-source: none;
+            border-style: solid;
+            border-width: 3px;
+            border-top-width: 1px;
+            border-radius: 8px;
+            border-color: var(--c-accent);
+          }
+        }
+      }
+    }
+
+    > aside {
+      --color-pf-secondary: var(--c-tertiary-dark);
+      background-image: url("../assets/theme/dark/dark_sidebar_top.webp"), url("../assets/theme/dark/dark_sidebar_bottom.webp");
+
+      &::before {
+        content: "";
+        float: left;
+        opacity: 1;
+        background-image: url("../assets/sf-logo.webp");
+        background-size: contain;
+        width: 234px;
+        height: 56px;
+        position: relative;
+        left: -0.5rem;
+        top: 0;
+        margin-top: 11px;
+        margin-bottom: 5px;
+      }
+
+      .sidebar {
+        select {
+          border: 1px solid var(--c-accent);
+          background-color: var(--c-accent-dark);
+        }
+        .data-value h2 {
+          color: var(--c-tertiary-light);
+        }
+        a.modifier {
+          color: var(--c-accent-light);
+          text-decoration: underline var(--c-accent-light);
+        }
+        button {
+          color: var(--c-emphasis);
+          background: var(--c-accent-dark);
+        }
+        .hitpoints .hp-big .container.current-hp {
+          background-image: linear-gradient(to right, var(--color-bg-trait-dark), var(--c-primary-dark), var(--color-bg-trait-dark));
+        }
+        .roll-data a.modifier.hover {
+          text-decoration: underline var(--color-pf-tertiary-darker);
+        }
+        .armor-class {
+          .ac {
+            filter: hue-rotate(65deg) brightness(1.6);
+          }
+          .shield.hp {
+            filter: hue-rotate(225deg) brightness(1.6);
+          }
+        }
+      }
+
+      img.logo {
+        display: none;
+      }
+    }
+  }
+}
```

```suggestion
}

```

Yeah, these are a pain🙃